MediaGalleryUi/Test/Mftf/ActionGroup/SearchStandaloneMediaGalleryAdminDataGridByKeywordActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="SearchStandaloneMediaGalleryAdminDataGridByKeywordActionGroup" extends="SearchAdminDataGridByKeywordActionGroup">
```

Comment:
I don't think there is a possibility of standalone media gallery search being different from embedded media gallery.

The idea is to keep them equal apart from specific actions that require context.

I think we can simply go with `SearchAdminDataGridByKeywordActionGroup`

Reply:
Initially, I tried with `SearchAdminDataGridByKeywordActionGroup` action group but as you can see the `type` is not defined for the `keyword` argument so I need to extend and the add the `type` attribute in order to run the test successfully

Reply:
I see, thanks @konarshankar07 ! I think it would be good to update the action group in magento repo as well